rules/type-declaration/tests/Rector/Identical/FlipTypeControlToUseExclusiveTypeRector/Fixture/skip_no_doc.php.inc

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace Rector\TypeDeclaration\Tests\Rector\Identical\FlipTypeControlToUseExclusiveTypeRector\Fixture;
+
+use stdClass;
+
+class SkipNoDoc
+{
+    public function run()
+    {
+        $stdClass = $this->getStdClass();
+        if ($stdClass === null) {
```

Comment:
This case should be covered too.
If `getStdClass()` returns `null|SomeObject` it should be processed the same way.

I'll merge it as it is, so you can add in next PR with less code to worry about

Reply:
I will try.